aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -875,12 +876,17 @@ impl ChangeSetInterface for VMChangeSet {
         &'a mut self,
         executor_view: &'a dyn ExecutorView,
         _module_storage: &'a impl AptosModuleStorage,
+        fix_prev_materialized_size: bool,
     ) -> impl Iterator<Item = PartialVMResult<WriteOpInfo>> {
-        let resources = self.resource_write_set.iter_mut().map(|(key, op)| {
+        let resources = self.resource_write_set.iter_mut().map(move |(key, op)| {
```

Comment:
curious, these are &mut right, what does move do here?

Reply:
I think `fix_prev_materialized_size` is moved?